financialaid/templatetags/financial_aid_tags.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
We decided to do this because it prevents the need to iterate through all the returned database objects again to attach the extra information.  Django's .annotate is sufficient to cover removing the use of subtract(), but it won't be able to replace the get_attribute().

I'm happy to redo the implementation to avoid the need for templatetags, but this is probably better in terms of performance.  Thoughts?